### DISCUSSION TITLE
How to connect EBYTE CMT2300A Dev Board to ESP32

### DISCUSSION THREAD

**Discussion by harmstorf** [tbnobody/OpenDTU#1092] [Q&A]
Dear community,

I am failing to operate an EBYTE E49-400/900MBL-01 (CMT2300A dev board) connected to an ESP32. The LEDs of both boards are lit, but OpenDTU cannot establish a connection to the radio module.

I used this pin mapping (E49-400 -> ESP32):
5 -> GND
8 -> +3.3 V
9 -> 4 (GPIO 4)
10 -> 5 (GPIO 5)
11 -> 23 (GPIO 23)
12 -> 18 (GPIO 18)

![S505fe0afd5c04efe89052c6e28d21b38F](https://github.com/tbnobody/OpenDTU/assets/648570/fe4c68cb-7ba7-4543-93d8-ce4274e45cd3)

Any ideas?

**Comment by Solarteur** [comment]
Did you see the hints here?
https://github.com/tbnobody/OpenDTU/discussions/902#discussioncomment-5978998

**Reply by harmstorf** [reply]
Thanks for the hint!

I have adopted the pin usage/configuration but am still out of luck: chip is configured but not connected.

![IMG_3039](https://github.com/tbnobody/OpenDTU/assets/648570/79a3c8fd-0e36-48a7-844a-7caf74bab75e)

**Reply by Daezu** [reply]
[This comment](https://github.com/tbnobody/OpenDTU/discussions/902#discussioncomment-13404131) works!
